src/libslic3r/Print.cpp

```diff
@@ -1240,7 +1240,7 @@ StringObjectException Print::validate(StringObjectException *warning, Polygons*
 #if 0
                 if (slicing_params0.gap_object_support != slicing_params.gap_object_support ||
                     slicing_params0.gap_support_object != slicing_params.gap_support_object)
-                    return  {("The prime tower is only supported for multiple objects if they are printed with the same support_top_z_distance"), object};
+                    return  {L("The prime tower is only supported for multiple objects if they are printed with the same support_top_z_distance"), object};
```

Comment:
I guess it should be `_L` here instead?

Reply:
I actually don't know when to choose `L()`, `_L()` or `_u8L()`.  The other return statements on this method are all `L()`, so I used it too.  Please advise if I should change the others too.

Reply:
both L() and _L() are supported. 
Please refer to run_gettext.bat/sh file.
`.\tools\xgettext.exe --keyword=L --keyword=_L --keyword=_u8L --keyword=L_CONTEXT:1,2c --keyword=_L_PLURAL:1,2 `